src/core/link-to-dfn.js

```diff
@@ -180,14 +176,12 @@ export async function run(conf) {
 
 function showLinkingError(elems) {
   elems.forEach(elem => {
-    elem.classList.add("respec-offending-element");
-    elem.title = "Linking error: not matching <dfn>";
-    pub(
-      "warn",
+    showInlineWarning(
```

Comment:
We might need to collect these warnings together maybe.

![image](https://user-images.githubusercontent.com/8426945/49722054-cb84d780-fc89-11e8-8866-b321418adaa3.png)
See 2 warnings for two `<a>baz</a>` which should be warned together.

Reply:
I don't feel strongly about the grouping. As an editor, as long as I can quickly get to what I need to fix, then I'm happy.

Reply:
I think the suggested behavior is more complex to write, as it mixes the message part and the linking part. Maybe a separate PR would be good.

Reply:
Yeah, agree (new PR). Let's see how this goes first :)